docs/making-your-own.md

```diff
@@ -1,39 +1,26 @@
 # Making your Own
 
-This project generates [operating system images](images.md) that you can use on your own PC.
-However, one of the main goals is to provide a custom toolkit for you to make your own custom image. 
+[Universal Blue](https://universal-blue.org/) is both a collection of native container images based on Fedora and a toolkit for you to make your own.
 
 !!! Note "This is moving fast"
 
     It's still early in the project's life, we're in need of people to help with these docs, so if you find a problem please consider [submitting a pull request](https://github.com/ublue-os/website) to help us improve.
 
 ## Why make your own image?
 
-The concept around image based operating systems balances on the idea that the core image is ready to go, ideally users don't need to touch it.
+The concept around image based operating systems balances on the idea that the core image is ready to go, ideally users don't need to touch it.  
 However, people like to tinker, so instead of [layering packages](https://docs.fedoraproject.org/en-US/iot/adding-layered/) that compromise the integrity of upgrades, the idea is to just make your own image so that you can just have a pristine setup over time for the life of the hardware.
 
 ### Reasons to make your own image
-- You want the reliability of image based operating systems like ChromeOS or Android but want more flexibility
+- You want the reliability of image based operating systems, but want to customize them to your liking
     - A complete Fedora experience with all the benefits of your preferences shipped out of the box
     - Ship your own terminal CLI experience on whatever distribution you like
 - Draw "outside the lines" of what Fedora provides in a repeatable manner
     - Make your own custom image without the overhead or responsibility of "making your own distro"
-    - But still be able to ingest all the improvements from Fedora every day
+    - Retain the ability to ingest all the improvements from Fedora and uBlue every day
 - You might need a specific kernel or module to get your hardware to work
 - It's fun
 
-## Should I use [`ublue-os/main`](https://github.com/ublue-os/main) or [`ublue-os/startingpoint`](https://github.com/ublue-os/startingpoint)?
```

Comment:
I agree with the removal of this. People should be using startingpoint. I actually haven't seen anyone build with multiple DE's but we can do this out of startingpoint with some modification